modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/fragment/renderer/order_actions/page.jsp

```diff
@@ -0,0 +1,21 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/fragment/renderer/order_actions/init.jsp" %>
+
+<div>
+	<react:component
+		module="{OrderActions} from commerce-order-content-web"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"isOpen", open
```

Comment:
should this `open` variable be named `isOpen`?

Reply:
There is a SF rule that enforces this, I cannot rename it.